edustripetech/edustripe_client_v1
# [chore] - setup app boilerplate with webpack

#### What does this PR do?
- Sets up webpack

#### How should this be manually tested?
- Clone the repo
- run `npm install`
- run `npm run dev` for `dev` environment
- run `npm run build` to bundle for production
- run  `npm start` to run bundled code on a local server. (Ensure to run build first).

#### Any background context OR Extra detail?
N/A

#### What are the relevant PT stories?
https://www.pivotaltracker.com/story/show/172003796

#### Screenshot
![Screenshot from 2020-03-30 02-26-26](https://user-images.githubusercontent.com/43539944/77867152-ecca5800-722d-11ea-9061-d339816bbc86.png)
